### DISCUSSION TITLE
vigne

### DISCUSSION THREAD

**Discussion by atygeorges** [Courseplay/Courseplay_FS22#2394] [Q&A]
bonjour, j'ai un pb cp aves la vigne. En effet cp traite pas l'ensemble du champ uniquement environ 6 à 8 lignes.
valable pour l'ensemble des outils.
merci de m'aider!
Cordialement

**Comment by Tensuko** [comment]
https://github.com/Courseplay/Courseplay_FS22/issues/1683#issuecomment-1147388739

**Reply by atygeorges** [reply]
vu réponse je vais tester!
merci
Cordialement

**Comment by Tensuko** [comment]
@Lactic68
you may can help out here?
This guy and someone else keep open new Issues with nothing in there and keep ask the same question.
I keep delete those Issues, you may explain them what to do and what exactly is the problem now?
As the answer about vine is answered in the comment I linked here...

**Comment by Lactic68** [comment]
Bonsoir @atygeorges ,

Comme expliqué par le précédent Ticket #1683 , il n'est actuellement pas possible pour le CP de créer des courses dans les vignes selon des bords trop complexes.

@Tensuko 
Happy to help. This issue seems to be about the merging of two fields which worked fine in CP with custom fields but doesn't work for wine. I have notify Atygeorges that this issue will not be resolved at the moment due to too complex borders issues with wine.

**Reply by Tensuko** [reply]
Thank you a lot for helping out! :)

**Comment by atygeorges** [comment]
merci pour la reponse
ordialement
